src/types.ts

```diff
@@ -58,6 +58,14 @@ export interface InputOptions {
   name: string;
   validation?: OptionsValidation;
   validators?: (Validator | ValidatorTuple)[];
+  /**
+   * Use this option to provide a validation message that applies
+   * to _all_ validators.
+   *
+   * **WARNING** using this option will override any messages supplied
+   * alongside individual validators.
+   */
+  validationMessage?: any;
```

Comment:
Just wanted to check with you @Madebyfen that this is ok to have an `any` type?

Reply:
Yeah so a couple of "big ticket" items before this hits v1 is getting types sorted wherever there are `any`s -- currently this includes inputs and validation messages.

Validation messages are supposed to be really flexible to make internationalisation a bit easier, but I've been thinking recently that they should be something like `ReactNode` which means they can be strings or components etc.

This definitely needs to be sorted, but as a separate issue.